gui/src/loader.rs

```diff
@@ -270,26 +277,72 @@ impl Loader {
                 self.daemon_started = false;
                 Command::none()
             }
+            Message::None => Command::none(),
             _ => Command::none(),
         }
     }
 
     pub fn subscription(&self) -> Subscription<Message> {
-        if let Some(Some(bitcoind_stdout)) =
-            self.internal_bitcoind.as_ref().map(|b| b.stdout.clone())
-        {
-            iced::subscription::unfold(0, bitcoind_stdout, move |stdout| async {
-                let msg = {
-                    let mut s = stdout.lock().await;
-                    let mut s = std::io::BufReader::new(s.deref_mut());
-                    let mut buffer = String::new();
-                    match s.read_line(&mut buffer) {
-                        Err(e) => Message::BitcoindLog(Some(e.to_string())),
-                        Ok(_) => Message::BitcoindLog(Some(buffer)),
+        if self.internal_bitcoind.is_some() {
+            let log_path = internal_bitcoind_debug_log_path(&self.datadir_path, self.network);
+            iced::subscription::unfold(
+                0,
+                (log_path, std::time::Instant::now()),
+                move |(log_path, last_poll)| async move {
+                    // Reduce the io load.
+                    if last_poll.elapsed() < Duration::from_millis(500) {
```

Comment:
This can of course be adapted, i went for a minimal change from the pretty fast update we had before. But IMO even updating only every 5 seconds would be more than enough.